plugin/pkg/admission/defaultingressclass/admission.go

```diff
@@ -104,7 +105,7 @@ func (a *classDefaulterPlugin) Admit(ctx context.Context, attr admission.Attribu
 	if !a.defaultIngressClassEnabled {
```

Comment:
can drop InspectFeatureGates and `defaultIngressClassEnabled` field since the feature is enabled now